conda-store-server/conda_store_server/app.py

```diff
@@ -297,6 +297,13 @@ def _docker_base_image(build: orm.Build):
         config=True,
     )
 
+    post_update_environment_build_hook = Callable(
+        default_value=None,
+        help="callable function taking conda_store and `orm.Environment` object as parameters to configure the update environment build task",
```

Comment:
Maybe we can make it a bit more clear exactly when this will be run...

```suggestion
        help="callable function taking conda_store and `orm.Environment` object as input arguments. This function can be used to add custom behavior that will run after an environment's current build changes.",
```